build-zlib_x64.cmd

```diff
@@ -0,0 +1,28 @@
+CALL "F:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" amd64
+
+SET var=%cd%
+ECHO %var%
+
+mkdir lib
+mkdir lib\x64
+
+cd zlib
+
+
+
+git reset --hard HEAD
+git checkout master
+
+git reset --hard origin/master
+
+git clean -f
+
+git apply ../0000-Statically-link-Zlib.patch --ignore-whitespace
```

Comment:
Can you move this into the main script where the other patches are applied? Best to keep this down to just building zlib.